src/dcqc/file.py

```diff
@@ -109,6 +109,8 @@ def get_file_type(cls, file_type: str) -> FileType:
 FileType("OME-TIFF", (".ome.tif", ".ome.tiff"), "format_3727")
 FileType("TSV", (".tsv"), "format_3475")
 FileType("BAM", (".bam"), "format_2572")
+FileType("FASTQ", (".fastq"), "format_1930")
```

Comment:
Ohh, my apologies, I see what you meant by the OME-TIFF example now.

Were you thinking of this originally?

```
FileType("FASTQ", (".fastq", ".fastq.gz")...
```

Reply:
Yeah, that's what I was initially thinking. That way might make more sense, but we would need to handle the zipped case eventually (unzip to run some tests) while also comparing the correct MD5 Checksum (zipped vs. unzipped) to the provided value. I.e. if someone provides the zipped MD5 sum we would need to run that check before unzipping the file.

Reply:
Makes sense to me.  Let's make change and we can open up a ticket to deal with zipped files. Im sure we will run into this issue with other larger file formats (.tar.gz, gz, etc)

Can you confirm that to the md5checksum test works with an actual gunzipped file?

Reply:
I made the changes discussed in this thread in the latest commit.